client/server/boot/index.js

```diff
@@ -25,6 +26,10 @@ export default function setup() {
 	app.use( userAgent.express() );
 	app.use( loggerMiddleware() );
 
+	if ( process.env.USE_PROFILER ) {
+		app.use( profilerMiddleware() );
```

Comment:
Should we `require()` `profilerMiddleware` only if `USE_PROFILER` is enabled?

Reply:
Yeah, that's a great idea!

Reply:
Done!